webhook/project.yml

```diff
@@ -28,5 +28,6 @@ packages:
           WEBHOOK_SAMUS_RETURNS_DEV: "${WEBHOOK_SAMUS_RETURNS_DEV}"
           WEBHOOK_CAVE_STORY_DEV: "${WEBHOOK_CAVE_STORY_DEV}"
           WEBHOOK_FACTORIO_DEV: "${WEBHOOK_FACTORIO_DEV}"
+          WEBHOOK_HUNTERS_DEV: "${WEBHOOK_HUNTERS_DEV}"
```

Comment:
I dont see `WEBHOOK_HUNTERS_DEV` defined neither in this repo nor the orga.
https://github.com/randovania/digital-ocean-app/settings/secrets/actions

Reply:
I added this anyway because I wasn't sure what else I needed to have changed.

Reply:
The variable needs to be set in digital ocean's page

Reply:
That webhook name is wrong anyways. It must be `WEBHOOK_MP_HUNTERS_DEV` because it is created by 
```
channel_enviroify = channel.upper().replace("-", "_")
webhook = os.environ[f"WEBHOOK_{channel_enviroify}"]
```
where  `mp-hunters-dev` is the used channel name here. 

On a side note this should be refactored in the future because the "channel name" is actually the webhook name and, as you can see, doesn't match with the current names for a lot of channels anymore.